plugins/openshift-developer/skills/address-review-pr/scripts/check_authorized.py

```
@@ -0,0 +1,205 @@
+]
+
+IGNORED_ACCOUNTS = [
+    "openshift-ci-robot",
+    "openshift-ci",
+    "openshift-merge-robot",
+    "openshift-bot",
+]
+
+
+def run_gh(args: list[str]) -> str:
+    result = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)